src/Pos/Launcher/Scenario.hs

```diff
@@ -66,8 +66,10 @@ runNode' plugins' = ActionSpec $ \vI sendActions -> do
                         ", address: "%build%
                         ", pk hash: "%build) pk addr pkHash
 
-    genesisStakeholders <- genesisStakeholdersM
-    logInfo $ sformat ("Genesis stakeholders: " %listJson) genesisStakeholders
+    genesisStakeholders <- view (lensOf @GenesisWStakeholders)
+    logInfo $ sformat ("Dust threshold: "%build)
+        (bootDustThreshold genesisStakeholders)
+    logInfo $ sformat ("Genesis stakeholders: " %build) genesisStakeholders
```

Comment:
I propose to print genesis stakeholders only if there are at most, say, 10 stakeholders. Otherwise only print number of stakeholders or something.